File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/platform/ecs/prx/serviceDiscovery/PrxConstants.java
/* Copyright (c) Koninklijke Philips N.V., 2018
 * All rights are reserved. Reproduction or dissemination
 * in whole or in part is prohibited without the prior written
 * consent of the copyright holder.
 */
package com.philips.platform.ecs.prx.serviceDiscovery;

public class PrxConstants {
	
	public static final String PRX_NETWORK_WRAPPER = "PRXNetworkWrapper";
	public static final String PRX_REQUEST_MANAGER = "PRXRequestManager";

	public enum Catalog {

		DEFAULT,

		CONSUMER,

		NONCONSUMER,

		CARE,

		PROFESSIONAL,

		LP_OEM_ATG,

		LP_PROF_ATG,

		HC,

		HHSSHOP,

		MOBILE,

		EXTENDEDCONSENT

	}

	public enum Sector {

		DEFAULT,

		B2C,

		B2B_LI,

		B2B_HC

	}

}

File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/platform/ecs/microService/prx/PrxConstants.kt
/*
 *  Copyright (c) Koninklijke Philips N.V., 2020
 *
 *  * All rights are reserved. Reproduction or dissemination
 *
 *  * in whole or in part is prohibited without the prior written
 *
 *  * consent of the copyright holder.
 *
 *
 */
package com.philips.platform.ecs.microService.prx

object PrxConstants {
    const val PRX_NETWORK_WRAPPER = "PRXNetworkWrapper"
    const val PRX_REQUEST_MANAGER = "PRXRequestManager"

    enum class Catalog {
        DEFAULT, CONSUMER, NONCONSUMER, CARE, PROFESSIONAL, LP_OEM_ATG, LP_PROF_ATG, HC, HHSSHOP, MOBILE, EXTENDEDCONSENT
    }

    enum class Sector {
        DEFAULT, B2C, B2B_LI, B2B_HC
    }
}